com.unity.multiplayer.mlapi/Runtime/Core/NetworkedObject.cs

```diff
@@ -407,14 +407,6 @@ public void Spawn(Stream spawnPayload = null, bool destroyWithScene = false)
             }
         }
 
-        /// <summary>
```

Comment:
nit: as in NetworkedBehavior, it simplifies the review to keep these types of changes separate